webapp/channels/src/components/drafts/drafts.tsx

```diff
@@ -65,32 +104,38 @@ function Drafts({
                     defaultMessage: 'Any messages you\'ve started will show here',
                 })}
             />
-            <div className='Drafts__main'>
-                {drafts.map((d) => (
-                    <DraftRow
-                        key={d.key}
-                        displayName={displayName}
-                        draft={d}
-                        isRemote={draftRemotes?.[d.key]}
+
+            <Tabs
+                id='draft_tabs'
+                activeKey={activeTab}
+                mountOnEnter={true}
+                unmountOnExit={false}
+                onSelect={handleSwitchTabs}
+            >
+                <Tab
+                    eventKey={0}
+                    title='Drafts'
+                    unmountOnExit={false}
+                    tabClassName='drafts_tab'
+                >
+                    <DraftList
+                        drafts={drafts}
                         user={user}
+                        displayName={displayName}
+                        draftRemotes={draftRemotes}
                         status={status}
```

Comment:
I feel here we are just prop drilling. Shouldn't this component be "one level higher" and the index file be assigned to the DraftList component, instead of to this one? That way this tab will look similar to the other tab (a component without props).

Reply:
No action needed here as discussed as we'd need the props here later on.